service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ChildControllerTest.java

```diff
@@ -111,9 +137,10 @@ void shouldReturnNoDateOfBirthErrorsWhenCaseDataIsEmpty() throws Exception {
                 .build());
     }
 
-    private AboutToStartOrSubmitCallbackResponse makeRequest(CallbackRequest request) throws Exception {
+    private AboutToStartOrSubmitCallbackResponse makeRequest(CallbackRequest request, String endpoint)
+        throws Exception {
```

Comment:
As there are changes to this class maybe we could use  AbstractControllerTest

Reply:
I have seen there is another PR for this, so will leave